app/models/order_detail.rb

```diff
@@ -74,6 +74,8 @@ def current_journal_rows
   end
 
   delegate :journal_date, to: :journal, allow_nil: true
+  delegate :reference, to: :journal, prefix: true, allow_nil: true
+
   def statement_date
```

Comment:
I considered trying to move some of these methods around since instance methods seemed mixed up in `delegates`, but in really these are essentially delegations where we want to use a different name.